app/grandchallenge/algorithms/templates/algorithms/job_form_create.html

```diff
@@ -0,0 +1,77 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+{% load url %}
+{% load get_obj_perms from guardian_tags %}
+{% load bleach %}
+{% load random_encode %}
+{% load static %}
+
+{% block title %}
+    Try-out Algorithm - {{ block.super }}
+{% endblock %}
+
+{% block script %}
+    {{ block.super }}
+    {{ form.media }}
+{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a
+                href="{% url 'algorithms:list' %}">Algorithms</a>
+        </li>
+        <li class="breadcrumb-item"><a
+                href="{{ algorithm.get_absolute_url }}">{{ algorithm.title }}
+        </a></li>
+        <li class="breadcrumb-item active"
+            aria-current="page">Try-out Algorithm
+        </li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+    {% get_obj_perms request.user for algorithm as "algorithm_perms" %}
```

Comment:
These permissions appear to be unused in this template - remove, right?

Reply:
Nice find